catalog/cataloger_commit_test.go

```diff
@@ -97,6 +95,9 @@ func TestCataloger_Commit(t *testing.T) {
 				t.Errorf("Commit() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+			if got != nil {
+				got.CreationDate = got.CreationDate.Truncate(time.Minute)
```

Comment:
This one will fail in some cases - skip the check my assign the `want` value or time default value for both.
The previous version we used the Truncate trick to reset the 'monotonic clock' value.

Reply:
Replaced with code that tests roughly what we want: that times are close to one another.  Yes, it's flaky when clocks drift apart, but OTOH we're running a newly-started Postgres container -- even on MacOS the time skew will be minimal.